appinfo/info.xml

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0"?>
 <info>
-    <id>contactsrework</id>
+    <id>contacts</id>
     <name>Contacts</name>
     <description>The new and improved app for your Contacts.</description>
     <licence>AGPL</licence>
     <author>Hendrik Leppelsack</author>
     <version>0.0.1</version>
```

Comment:
Since the new contacts app has no migrations this is pointless from my poc.

But we should still adopt this properly.

Let's name it 1.0.0.0